sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -572,15 +572,14 @@ class CertificatePolicy(object):
         for RSA.
     :keyword bool reuse_key: Indicates if the same key pair will be used on certificate
         renewal.
-    :keyword curve: Elliptic curve name. For valid values, see KeyCurveName.
-    :paramtype curve: str or ~azure.keyvault.certificates.KeyCurveName
-    :keyword ekus: The enhanced key usages.
-    :paramtype ekus: list[str]
+    :keyword key_curve_name: Elliptic curve name. For valid values, see KeyCurveName.
+    :paramtype key_curve_name: str or ~azure.keyvault.certificates.KeyCurveName
+    :keyword enhanced_key_usage: Ways the enhanced key can be used
```

Comment:
There is no "enhanced key". This refers to extended ways the key of the certificate can be used. See https://tools.ietf.org/html/rfc5280#section-4.2.1.12.